azavea/raster-foundry
# Add grid endpoint and frontend

## Overview

Add grid endpoint
This endpoint takes a bounding box and returns geojson featurecollection containing a grid of polygons that lines up with the basemap tiles, where each grid square has the total number of scenes intersecting it

Add scene grid frontend

### Checklist

- [X] Upgrade Geotrellis to v1.0.0 when it's released

### Demo

![image](https://cloud.githubusercontent.com/assets/4392704/21029413/91146dc6-bd67-11e6-8f7c-6292a5a91657.png)


### Notes

Grid is limited to areas of the map that actually return results - if an area doesn't have any scenes, it will not have a cell. 

This will definitely require further optimizations down the line, probably something with geohashes.

## Testing Instructions

 * Open the browse view and wait for scenes to load. Sometimes the first grid takes a while to load since it fires off with the wrong bounding box. The more zoomed in the map is, the faster the queries will be.
* Mouse over a grid cell to see its scene count

Connects #653
